README.md

```diff
@@ -183,6 +183,23 @@ adb shell ime set com.google.android.inputmethod.latin/com.android.inputmethod.l
 ```
 
 
+## Notifications
+
+Since version 2.16.0 Appium Settings supports retrieval of system notifications.
+You need to manually switch the corresponding security switcher next to `Appium Settings` 
+application name in `Settings->Notification Access` (the path to this page under Settings 
```

Comment:
👍
Note for Android 9 (Android One, Nokia)
`Settings > App & Notifications > special app access > Notification access`

Reply:
On my Samsung it's different ;)

Reply:
Hello! Tell me please, how can I grant permission for get status bar notifications via adb? Is it available?